Packages/com.unity.inputsystem/InputSystem/Actions/InputActionReference.cs

```diff
@@ -159,6 +160,19 @@ public override string ToString()
             return base.ToString();
         }
 
+        private static string GetNiceName(InputAction action)
```

Comment:
Minor: Not sure whats so nice about it :) but maybe GetUserFriendlyName  or GetDisplayName is more conventional?

Reply:
Fixed